packages/kbn-test-jest-helpers/src/stub_broadcast_channel.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+const channelCache: BroadcastChannel[] = [];
+
+class StubBroadcastChannel implements BroadcastChannel {
```

Comment:
Our jest environment does not supply a native `BroadcastChannel` implementation, so we are forced to provide our own. This follows a similar convention to the stubbed WebWorker defined in `packages/kbn-test-jest-helpers/src/stub_web_worker.ts`